### DISCUSSION TITLE
How to properly restart a server?

### DISCUSSION THREAD

**Discussion by MaxVerevkin** [williamboman/nvim-lsp-installer#545] [Q&A] [answered]
Everything works perfectly fine, but if I run `:LspRestart`, I get the following error in `:LspInfo`:

```
cmd is executable: Unable to find executable. Please check your path and ensure the server is installed
```

I use `rust_analyzer`.

**Comment by williamboman** [comment]
Hello! Do you see any errors in the LSP logs (`:exe 'tabnew ' .. luaeval("vim.lsp.get_log_path()")`)?

**Reply by MaxVerevkin** [reply]
Hi. No, there are no errors in the log file.

**Comment by MaxVerevkin** [comment, marked as answer]
The new way of initializing servers fixed it.
